test_simplified_configs.py

```diff
@@ -0,0 +1,162 @@
+from typing import List, Optional
+
+from pydantic import BaseModel, ValidationError
+
+from ert3.config import PluginConfigManager
+from ert3.config import ErtBaseModel, ErtPluginField
+
+
+class _StagesConfig(BaseModel):
+    class Config:
+        validate_all = True
+        validate_assignment = True
+        extra = "forbid"
+        allow_mutation = False
+        arbitrary_types_allowed = True
+
+
+class TransformationConfigBase(_StagesConfig):
+    """Common config for all Transformations"""
+
+    location: str
+
+
+class FileTransformationConfig(TransformationConfigBase):
+    mime: str = ""
+
+
+class SummaryTransformationConfig(TransformationConfigBase):
+    smry_keys: Optional[List[str]] = None
+
+
+class DirectoryTransformationConfig(TransformationConfigBase):
+    pass
+
```

Comment:
Shouldn't we call the same as Transformations themselves; eg. `TarRecordTransformation`? Also I guess that executable transformation won't be configured (but only adopted from command then)?

Reply:
If this is the direction we choose to go for, this would be named according to the transformation that already exist. For now this is not meant to be an exact, nor exhaustive, list of the available transformations :slightly_smiling_face: